langaratribe/src/Pages/Home/HomeComponents/Circles.js

```diff
@@ -1,15 +1,24 @@
+// Circles component for icon circles
+// Need to find out a way to return correct icons depending on the parameter passed through obj literal
+import { SiGoogleclassroom, SiAirtable, SiAlgolia, SiAmazons3, SiAndela } from "react-icons/si";
 
-const Circles = () => {
-    
-    const icon = "";
-
-    if(props.)
+const Circles = (props) => {
 
     return (
-        <div id="course"
-            onMouseEnter={() => props.setState("course")}
-            onMouseLeave={() => props.setState("default")}
-        ></div>
+        <div className="iconFlex">
+            <div id={props.id}
+                onMouseEnter={() => props.setState(`${props.id}`)}
```

Comment:
onMouseEnter= {() => {
props.setState(`${props.id}`);
String iconName = props.id
return(
<div>
<{iconName}/>
</div>
)
}
}


.

Reply:
I want to look at the full useState statement too

Reply:
Really good job Jay. You are doing really well.